public/video-ui/src/pages/Upload/index.jsx

```diff
@@ -70,7 +70,7 @@ class VideoUpload extends React.Component {
               activeVersion={activeVersion}
               s3Upload={this.props.s3Upload}
               uploads={this.props.uploads}
-              deleteAsset={this.props.videoActions.deleteAsset}
+              deleteAsset={this.props.videoActions.deleteAssets}
```

Comment:
Just check, is this right? I can't see `deleteAssets` in the `mapDispatchToProps`

Reply:
Ah gotcha! For what it's worth that really confused me because effectively we have `deleteAsset.deleteAsset` and `deleteAsset.deleteAssets`. Can we update the imports to make this clearer?

Reply:
That sounds good to me.  

While we can just import the new `deleteAssets`, I am a bit uncertain about deviating from the pattern of importing all exported functions from those modules.  Worried that future developers would assume to have all exported functions available, especially when JS don't have type checking.

So for now, I just change the import name as `allDeleteAssetActions`, which is an improvement anyway.